conda-store-server/conda_store_server/_internal/server/views/api.py

```diff
@@ -133,6 +134,37 @@ def paginated_api_response(
     }
 
 
+def deprecated(sunset_date: datetime.date):
```

Comment:
Missing return type annotation, but I don't think this should be a blocker here.